vsts/generate_javadocs_branch.ps1

```diff
@@ -48,53 +53,53 @@ function CreateJavadocReleaseBranch($GitHubName, $GitHubEmail, $Sources) {
 
     Write-Host "Copying generated javadocs to replace current javadocs"
     Set-Location apidocs
-    Remove-Item ..\deps -Force -Recurse
-    Copy-Item -Force -Path .\deps\* -Destination ..\deps
-    Copy-Item -Recurse -Force -Path .\deps\com -Destination ..\deps
+    Remove-Item ..\$folderName\deps -Force -Recurse
```

Comment:
Just realized, I need to add a check to see if the folder exists or not. I will update the script. 'Preview' folder does not exist today.